include/mbedtls/pk.h

```diff
@@ -509,34 +509,6 @@ int mbedtls_pk_copy_from_psa(mbedtls_svc_key_id_t key_id, mbedtls_pk_context *pk
  */
 int mbedtls_pk_copy_public_from_psa(mbedtls_svc_key_id_t key_id, mbedtls_pk_context *pk);
 
-/**
- * \brief           Verify signature.
- *
- * \note            The signature algorithm used will be the one that would be
- *                  selected by \c mbedtls_pk_get_psa_attributes() called with a
- *                  usage of #PSA_KEY_USAGE_VERIFY_HASH - see that function's
- *                  documentation for details.
- *                  If you want to select a specific signature algorithm, see
- *                  \c mbedtls_pk_verify_ext().
- *
- * \note            This function currently does not work on RSA keys created
- *                  with \c mbedtls_pk_wrap_psa().
- *
- * \param ctx       The PK context to use. It must have been populated.
- * \param md_alg    Hash algorithm used.
- * \param hash      Hash of the message to sign
- * \param hash_len  Hash length
- * \param sig       Signature to verify
- * \param sig_len   Signature length
- *
- * \return          0 on success (signature is valid),
- *                  #PSA_ERROR_INVALID_SIGNATURE if the signature is invalid,
- *                  or another specific error code.
- */
-int mbedtls_pk_verify(mbedtls_pk_context *ctx, mbedtls_md_type_t md_alg,
```

Comment:
We can' remove public functions! As I commented elsewhere, the task was ill-defined, but it only applies to private functions. Public functions have to stay until the next major release.